packages/nextjs/components/scaffold-eth/Contract/ReadOnlyFunctionForm.tsx

```diff
@@ -51,12 +51,13 @@ export const ReadOnlyFunctionForm = ({ contractAddress, abiFunction }: TReadOnly
     <div className="flex flex-col gap-3 py-5 first:pt-0 last:pb-1">
       <p className="font-medium my-0 break-words">{abiFunction.name}</p>
       {inputElements}
-      <div className="flex justify-between gap-2">
-        <div className="flex-grow">
+      <div className="flex justify-between gap-2 flex-wrap">
+        <div className="flex-grow w-4/5">
```

Comment:
Had to add `w-4/5` for `break-words` work 

Also added `flex-wrap` so that even if there is an edge case the `read` button doesn't go out of the parent div : 


https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/d5629101-8e61-4357-9c4d-0c4b09937c19